sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1125,6 +1125,24 @@ pi_result _pi_platform::initialize() {
   ZeDriverApiVersion = std::to_string(ZE_MAJOR_VERSION(ZeApiVersion)) + "." +
                        std::to_string(ZE_MINOR_VERSION(ZeApiVersion));
 
+  // Cache driver extension properties
+  uint32_t Count = 0;
+  ZE_CALL(zeDriverGetExtensionProperties, (ZeDriver, &Count, nullptr));
+
+  if (Count == 0) {
```

Comment:
Why do you think we need to give error in this case?

Reply:
would you remove this?

Reply:
I kept it as it was in Gail's patch. My reason for keeping it is to do an early return and saving the overhead of the next zeDriverGetExtensionProperties call.

Reply:
it made sense when it was looking for "global-offset" specifically" but not when it is made a general query

Reply:
removing it.